Ifeanyi/Chapter_2/exercise_2.14.1

```diff
@@ -0,0 +1,13 @@
+
+a='All'
+b='work'
+c='no'
+d='play'
+e='makes'
+f='jack'
+g='a'
+h='dull'
+i='boy'
+print(a+' '+b+' '+c+' '+d+' '+e+' '+f+' '+g+' '+h+' '+i)
```

Comment:
Doing this is quite not the best. Concatenating each letter with a space is doesn't look too good to me but it solves the problem and we'll see what happens as the course progresses :)